.mvn/maven.config

```diff
@@ -0,0 +1,9 @@
+--batch-mode
+--errors
+--fail-at-end
+--show-version
+--no-transfer-progress
+--threads 2C
```

Comment:
I think Camunda cannot be built in parallel, trying to run it with `mvnd` also failed for me

Reply:
Could be. I have removed this now.
I think the problems are elsewhere, but I can't locate the issue so far.

Reply:
The problem was an outdated Maven wrapper.